openmsupply/open-msupply
# Fix tab label

Fixes #1459 

# 👩🏻‍💻 What does this PR do? 
 
The cause of the problem is that the `useTranslation` hook was not handling default values correctly.
While repairing that, I have refactored the `ModalTabs` usage slightly, as it was relying on the default value in an inelegant way. The implementation was translating twice and this isn't required.

<img width="520" alt="image" src="https://user-images.githubusercontent.com/9192912/228683550-5a781ef9-7168-40ff-92f3-f55964af0cfe.png">


### Offroads
- I found that having a logout in the CMD+K useful, and have also added `Admin`
- The `sed` command that was a temporary fix does not work on linux (different sed version) so have changed to perl which works on both platforms.
- Added the `.gitignore` entry for android assets

# 🧪 How has/should this change been tested? 

- Check the stockline edit modal and customer requisition modals, for the tab names.
- Check `yarn generate` for the perl command
- cmd+k for the new actions


## 📃 Documentation

- [ ] Extra cmd+k actions : can update the screenshot in the [tutorial](https://docs.msupply.foundation/docs/introduction/tutorial/)
